src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('ul li')];
+
+  list.sort((a, b) => {
+    const item1 = parseInt(a.dataset.salary.replace(/[\D]/g, ''));
+    const item2 = parseInt(b.dataset.salary.replace(/[\D]/g, ''));
+
+    return item2 - item1;
+  });
+
+  list.forEach((item) => {
+    document.querySelector('ul').appendChild(item);
+  });
+}
+
+function getEmployees() {
+  const list = [...document.querySelectorAll('ul')];
```

Comment:
This line is selecting all 'ul' elements, but it seems like you want to select 'li' elements instead, similar to the 'sortList' function. Make sure you are selecting the correct elements to retrieve employee data.